src/lib.rs

```diff
@@ -613,6 +613,44 @@ impl Client {
             )))))
         }
     }
+
+    /// Get data with cas and update the expiration time of an existing data by key
+    ///
+    /// If the key is found, `Some(Value)` is returned, describing the metadata and data of the key.
+    ///
+    /// Otherwise, [`Error`] is returned.
+    pub async fn gats<K>(&mut self, exptime: i32, key: K) -> Result<Option<Value>, Error>
```

Comment:
```suggestion
    pub async fn gats<K>(&mut self, key: K, exptime: i32) -> Result<Option<Value>, Error>
```

Let's keep the APIs consistent with `key` being the first argument.

Reply:
Great, I'll change it now!

Reply:
Ok, resolve `[year 2038 problem](https://en.wikipedia.org/wiki/Year_2038_problem)`